src/js/burials/helpers.jsx

```diff
@@ -1,8 +1,120 @@
 import React from 'react';
+import Raven from 'raven-js';
 import moment from 'moment';
-
+import environment from '../common/helpers/environment.js';
 import { transformForSubmit } from '../common/schemaform/helpers';
 
+function checkStatus(guid) {
+  const userToken = window.sessionStorage.userToken;
+  const headers = {
+    'Content-Type': 'application/json',
+    'X-Key-Inflection': 'camel',
+  };
+
+  if (userToken) {
+    headers.Authorization = `Token token=${userToken}`;
+  }
+  return fetch(`${environment.API_URL}/v0/burial_claims/${guid}`, {
+    headers,
+    mode: 'cors'
+  })
+    .then(res => {
+      if (res.ok) {
+        return res.json();
+      }
+
+      return Promise.reject(res);
+    }).catch(res => {
+      if (res instanceof Error) {
+        Raven.captureException(res);
+        Raven.captureMessage('vets_burial_poll_client_error');
+
+        // keep polling because we know they submitted earlier
+        // and this is likely a network error
+        return Promise.resolve();
+      }
+
+      // if we get here, it's likely that we hit a server error
+      return Promise.reject(res);
+    });
+}
+
+const POLLING_INTERVAL = 1000;
+
+function pollStatus({ guid, confirmationNumber, regionalOffice }, onDone, onError) {
```

Comment:
Opted to just pass the values from the previous calls to `pollStatus`. I think when we refactor this, I would like to have the the response built from the different calls in a single closure so it's easier to see that we're building a complete response from multiple calls.